src/renderer.rs

```diff
@@ -320,4 +325,71 @@ impl Renderer {
             line(color::BLACK.into(), 1., [start[0], start[1], end[0], end[1]], c.transform, g);
         }
     }
+
+    /// Save the drawings to SVG
+    fn save_svg<P: AsRef<path::Path>>(&mut self, path: P) {
+        let drawing_state = self.app.drawing();
+        let mut document = svg::Document::new()
+            .set("viewBox", (0, 0, drawing_state.width, drawing_state.height));
+
+        // set background color - https://stackoverflow.com/a/11293812/9276882
+        let bg_color = drawing_state.background;
+        let background = SvgRect::new()
+            .set("width", "100%")
+            .set("height", "100%")
+            .set("fill", format!("rgba({},{},{},{})", bg_color.red, bg_color.green, bg_color.blue, bg_color.alpha));
+
+        document = document.add(background);
+
+        let center = Point {x: drawing_state.width as f64 * 0.5, y: drawing_state.height as f64 * 0.5};
+
+        for drawing in &self.drawings {
+            match *drawing {
+                Drawing::Path(ref path) => {
+                    let &Path { start, end, ref pen } = path;
+                    let &Pen { thickness, color, enabled: _ } = pen;
+
+                    let start = start.to_screen_coords(center);
+                    let end = end.to_screen_coords(center);
+
+                    let line = SvgLine::new()
+                        .set("x1", start.x)
+                        .set("y1", start.y)
+                        .set("x2", end.x)
+                        .set("y2", end.y)
+                        .set("stroke", format!("rgba({},{},{},{})", color.red, color.green, color.blue, color.alpha))
```

Comment:
This suggestion is totally **optional**, but I noticed that you repeated the code to format a `Color` as an SVG color value a few times. If you'd like, you can remove that repetition using an [inner function](https://stackoverflow.com/questions/26685666/a-local-function-in-rust). Totally up to you. :smile: